public/demo.html

```diff
@@ -0,0 +1,429 @@
+
```

Comment:
What is the purpose of this file? (It seems useful, but seems like it might be dependent on a specific vector tile schema.)